docs/guide/blog/index.md

```diff
@@ -111,7 +128,7 @@ The `case *types.MsgCreatePost` statement handles messages of type `MsgCreatePos
 
 Every module has a handler function like this to process messages and call keeper methods.
 
-## Process Messages
+## Define Messages
```

Comment:
Messages were "defined" earlier, here we define the logic of message handlers.

Reply:
Lets say: 

## Define Messages Logic